src/odemis/driver/semcomedi.py

```diff
@@ -2594,13 +2594,16 @@ def __init__(self, name, role, parent, channels, limits, settle_time,
           (lower/upper limit in X) if magnification is 1 (in m)
         park (None or 2-tuple of (0<=float)): voltage of resting position,
           if None, it will default to top-left corner.
-        scanning_ttl (None or dict of int -> (bool or bool, str)): list of
-          digital output ports to indicate the ebeam is scanning or not.
-          If True, it is set to high when scanning, with False, the output is
-          inverted.
+        scanning_ttl (None or dict of int -> (bool or (bool, str), or (bool, str, bool))):
+          List of digital output ports to indicate the ebeam is scanning or not.
+          First boolean is "high_auto": if True, it is set to high when scanning,
+            with False, the output is inverted.
           If a str is provided, a VA with that name will be created, and will
-          allow to force the TTL to enabled (True) or disabled (False), with
-          None (auto) being the default.
+            allow to force the TTL to enabled (True) or disabled (False), with
+            None (auto) being the default.
```

Comment:
It would be nice if None is into parentheses, like the other 2 options (True, False).

Reply:
Updated.